instances/latest/terminologies/technique/differentialInterferenceContrastMicroscopy.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/differentialInterferenceContrastMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "An optical microscopy technique, used to enhance the contrast in unstained, transparent samples [taken from [Wikipedia](https://en.wikipedia.org/wiki/Differential_interference_contrast_microscopy)].",
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0739494",
```

Comment:
Was this discussion during our meeting? This is for IR-DIC video microscopy. It sounds more like a subclass of DIC microscopy, but if this was discussed it's ok.

Reply:
Yes, we discussed this: DIC will be combined with IR DIC video microscopy

Reply:
Yes, http://uri.interlex.org/base/ilx_0739494 is correct for this. There isn't really another type of DIC that is regularly used. If someone complains we can fix the mapping.